cwms_radar_api/src/test/java/cwms/radar/data/dao/ClobDaoTest.java

```diff
@@ -0,0 +1,303 @@
+package cwms.radar.data.dao;
+
+import static cwms.radar.data.dao.DaoTest.getDslContext;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.google.common.flogger.FluentLogger;
+import cwms.radar.api.errors.AlreadyExists;
+import cwms.radar.data.dto.Clob;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled
```

Comment:
why disabled? since it needs to talk to the database we could put it under the IT tests.

Reply:
It was Disabled b/c I didn't think to turn it into an IT.  Should I?